### ISSUE TITLE
`borderColor` is not overriding `border`

### ISSUE THREAD

**Issue by zamotany** [zamotany/react-slate#66]
When both `border` and `borderColor` properties are applied to `style` prop, `borderColor` has no effect (`borderColor` after `border`):

```js
<View
  style={{
    border: "solid white",
    borderColor: "red"
  }}
>
  Hello
</View>;
```

which makes it impossible to do:

```js
const defaultStyle = {
  border: "solid white"
};

<View
  style={{
    ...defaultStyle,
    borderColor: "red"
  }}
>
  Hello
</View>;
```

Probably the place to looks for is here: https://github.com/zamotany/react-slate/blob/9499d41416dc6764559bc69f387ed6a5f4b07176/packages/react-slate/src/utils/style.js#L120-L125

**Comment by sunilhari** [comment]
@zamotany 

Can you write a little description on this.

**Comment by zamotany** [comment]
@sunilhari updated description

**Comment by zamotany** [comment]
@sunilhari 
Whatever property was declared later takes precedence. Similar how spreading works.

* `borderColor`, then `border` -> `border` takes precedence:
  ```js
  { borderColor: 'white', border: 'solid red' } // => border color should be `red`
  ```

* `border`, then `borderColor` -> `borderColor` takes precedence:
  ```js
  { border: 'solid white', borderColor: 'red' } // => border color should be `red`
  ```

**Comment by jukben** [comment]
This is how it works in CSS I think that we should mirror the behavior 
```js
{ borderColor: 'white', border: 'solid red' } // => border color should be `red`
```

**Comment by jukben** [comment]
@zamotany Sorry I should have been more explicit. The code I have attached wasn't the citation just the example that I like more.

E.g this behaivor makes sense for me more

1) ```css
border-color: white;
border: 1px solid red;

// border: 1px solid red;
```
because it follow CSS specification.

2) ```css
margin-left: 0;
margin: 20px;

// margin-left: 20px; margin-right: 20px; margin-top: 20px; margin-bottom: 20px
```

**Comment by zamotany** [comment]
We are going to go with specificity in that case. It is how React Native does it and our API is based on both CSS and React Native. So considering that order based is more fragile and less robust the higher the specificity the more priority the property has, so the PR is fine.

**Comment by zamotany** [comment]
Fixed in #86 thanks to @mik639
